src/rules.ts

```diff
@@ -717,6 +733,64 @@ export const rules: Rule[] = [
         ),
       ],
   ),
+  new Rule(
+      'Convert Spaces to Tabs',
+      'Converts 4 spaces to 1 tab.',
+      RuleType.SPACING,
+      (text: string, options = {}) => {
+        const tabsize = String(options['tabsize']);
+        const tabsize_regex = new RegExp('^(\t*) {' + String(tabsize) + '}', 'gm');
+
+        while (text.match(tabsize_regex) != null) {
+          text = text.replace(tabsize_regex, '$1\t');
+        }
+        return text;
+      },
+      /* eslint-disable no-mixed-spaces-and-tabs, no-tabs */
+      [
+        new Example(
+            'Converting spaces to tabs with `tabsize = 3`',
+            dedent`
+            - text with no indention
+               - text indented with 3 spaces
+            - text with no indention
+                  - text indented with 6 spaces
+        `,
+            dedent`
+            - text with no indention
+            	- text indented with 3 spaces
+            - text with no indention
+            		- text indented with 6 spaces
+        `,
+            /* eslint-enable no-mixed-spaces-and-tabs, no-tabs */
+            {tabsize: '3'},
+        ),
+      ],
+      [
+        new Option('Tabsize', 'Number of Spaces that will be converted to a Tab (default: 4)', '4'),
```

Comment:
I forgot to implement IntegerOptions, do you mind if I push to your branch? I can also push it to master, and then you will have to merge again.

Reply:
no problem, you can simply push it to my branch, seems like less work

Reply:
or you could accept my PR into the master and then simply InterOptions to the master? Wouldn't that be even easier?